common/components/common/projects/AboutProjectDisplay.jsx

```diff
@@ -222,11 +222,13 @@ class AboutProjectDisplay extends React.PureComponent<Props, State> {
 
           <div className='AboutProjects-details'>
             <div id='project-details'>
-              {
-                project.project_description + "\n" +
-                project.project_description_solution + "\n" +
-                project.project_description_actions
-              }
+              {project.project_description}
+              <br></br>
```

Comment:
Actually, since Solution and Actions are optional fields, we should check to see that they are present before adding the padding between them.  Otherwise, we could end up putting in a bunch of blank space we don't need.